docs/source/usage/common_scenarios.rst

```diff
@@ -69,4 +69,52 @@ In order for the lessee to access a node on a private network, they can do one o
     openstack esi trunk add network --tagged-networks <private network> <trunk name>
     openstack esi node network attach --trunk <trunk name> <node>
 
+Using External Networks
+-----------------------
+
+The recommended way to allow external network access to a baremetal node is by `creating an external provider network`_. The instructions in the link explain how; in addition, this network should be created as ``--provider-network-type vlan`` with the appropriate vlan specified by ``--provider-segment``.
+
+Once that's done, a lessee can provision a node upon their provisioning network and then gain external network access using one of two methods.
+
+Floating IP
```

Comment:
Do OpenStack security groups play a part here when we actually want to reach the node? In the VM world we have to explicitly enable what ports are allowed for the VM.

Reply:
From my testing, it actually does not.